reanahub/reana-client-go
# cli: implement loglevel flag

closes https://github.com/reanahub/reana-client-go/issues/2

Implements a global `loglevel` flag by using a [Logrus](https://github.com/sirupsen/logrus) library.

To test:
```
- go run . du -w workflow -l DEBUG # should display the output from go-swagger and print all the flags used in the cmd
- go run . open -w workflow -l INFO # should display only an info level message
- go run . du -w workflow -l TEST # should return an error
```

As part of this PR changed `ValidateArgChoice()` to more generic `ValidateChoice()` to allow using it also for flags.
